scene/2d/camera_2d.cpp

```diff
@@ -409,6 +409,35 @@ void Camera2D::force_update_scroll() {
 	_update_scroll();
 }
 
+void Camera2D::reset_smoothing() {
+
+	smoothed_camera_pos = camera_pos;
+	_update_scroll();
+}
+
+void Camera2D::align() {
+
+	Size2 screen_size = get_viewport_rect().size;
+	screen_size=get_viewport_rect().size;
```

Comment:
yes, the whole logic behind align() is a duplicate of parts of get_camera_transform()